master/pkg/tasks/task_command.go

```diff
@@ -17,6 +17,8 @@ type genericCommandSpecMetadata struct {
 	PublicKey     *string `json:"publicKey"`
 	ExperimentIDs []int32 `json:"experiment_ids"`
 	TrialIDs      []int32 `json:"trial_ids"`
+	// CHAT: we want this to be part of the command snapshot that gets persisted.
+	WorkspaceID model.AccessScopeID `json:"workspace_id"`
```

Comment:
could be part of `Base TaskSpec` as well 🤔

Reply:
I think for technical purposes, there won't be a difference. For logical reasons, lumping it with metadata does make sense.

Wasn't there also a preference to use `ProjectID` instead of `WorkspaceID`?